main.js

```diff
@@ -12,11 +12,40 @@ import {parseRequest, parseSignatureHeader} from 'http-signature-header';
 import {CapabilityInvocation} from 'ocapld';
 import {TextDecoder, TextEncoder, base64Decode} from './util.js';
 
+/**
+ * Verifies a zcap in the form of an http-signature header.
+ *
+ * @param {object} options - Options to use.
+ * @param {URL} options.url - The url of the request.
+ * @param {string} options.method - The HTTP request method.
+ * @param {Array<string>} options.headers - The headers used to make the
+ *   signature.
```

Comment:
We don't yet know which headers were used to make the signature, right? These are just the headers we received in the request (is my understanding).

```suggestion
 * @param {Array<string>} options.headers - The headers from the request.
```